src/components/DataTable.js

```diff
@@ -29,6 +42,46 @@ export default class DataTable extends React.Component {
     window.removeEventListener('resize', this.resize);
   }
 
+  setSortState(col) {
+    const { sort } = this.state;
+
+    if (!sort.order) {
+      sort.order = 'asc';
+    }
+    if (sort.order && sort.column === col) {
+      sort.order = sort.order === 'asc' ? 'desc' : 'asc';
```

Comment:
I can update it if you think it's clearer, but it's setting the `column` as well as the `order`, so it's not just toggling (I think of a toggle as only having 2 options, and `column` has more than that).